aksel.nav.no/website/app/dev/(prinsipper)/prinsipper/[[...prinsipp]]/page.tsx

```diff
@@ -0,0 +1,80 @@
+import { Metadata } from "next";
+import { PortableTextBlock } from "next-sanity";
+import { notFound } from "next/navigation";
+import { Image } from "sanity";
+import { BodyLong, BodyShort, HStack, Heading, VStack } from "@navikt/ds-react";
+import { CustomPortableText } from "@/app/CustomPortableText";
+import { sanityFetch } from "@/app/_sanity/live";
+import { PRINSIPPER_BY_SLUG_QUERY } from "@/app/_sanity/queries";
+import { urlForOpenGraphImage } from "@/app/_sanity/utils";
+
+type Props = {
+  params: Promise<{ prinsipp: string[] }>;
+};
+
+export async function generateMetadata({ params }: Props): Promise<Metadata> {
+  const { prinsipp } = await params;
+
+  const { data: seoData } = await sanityFetch({
+    query: PRINSIPPER_BY_SLUG_QUERY,
+    params: { slug: `prinsipper/${prinsipp.join("/")}` },
+    stega: false,
+  });
+
+  return {
+    title: seoData?.heading,
+    description: seoData?.seo?.meta,
+    openGraph: {
+      images: urlForOpenGraphImage(seoData?.seo?.image as Image),
+    },
+  };
+}
+
+export default async function Page({ params }: Props) {
+  const { prinsipp } = await params;
+
+  const { data: pageData } = await sanityFetch({
+    query: PRINSIPPER_BY_SLUG_QUERY,
+    params: { slug: `prinsipper/${prinsipp.join("/")}` },
+  });
+
+  if (!pageData || !pageData.content || !pageData.heading) {
+    notFound();
+  }
+
+  const authors =
+    pageData?.contributors
+      ?.filter((auth) => !!auth.title)
+      .map((auth) => auth.title ?? "") ?? [];
+
+  return (
+    <div>
+      <Heading level="1" size="xlarge" data-aksel-heading-color spacing>
```

Comment:
We don't need to do anything about it in this PR, but should we consider making a small wrapper for the Heading component for use on the Aksel web page, that just adds `data-aksel-heading-color` (since it seems like it is used everywhere) and possibly `spacing` too? It is an easy thing to forget, so this would help with consistency.